app/cdap/components/WranglerV2/DataQualityCircularProgressBar/index.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core';
+import { lightGreen, red } from '@material-ui/core/colors';
+import { IGeneralStatistics } from 'components/GridTable/types';
+import { IHeaderNamesList } from 'components/WranglerGrid/SelectColumnPanel/types';
+import React from 'react';
+import SemiCircleProgressBar from 'react-progressbar-semicircle';
+import styled from 'styled-components';
+
+const GREEN_COLOR = lightGreen['400'];
+const RED_COLOR = red.A100;
+
+const SemiCircularProgressBarWrapper = styled(Box)`
+  width: 75px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 100%;
+  letter-spacing: 0.15px;
+`;
+
+const GreenSemiCircleProgressBar = styled(SemiCircularProgressBarWrapper)`
+  color: ${GREEN_COLOR};
+`;
+
+const RedSemiCircleProgressBar = styled(SemiCircularProgressBarWrapper)`
+  color: ${RED_COLOR};
+`;
+
+/**
+ * @param statistics is the column data received from apis, used to calculate the data quality of the respective column
+ * @param  columnList is the list of all column headers for which we calculate the data quality
+ * @return the list of column's data quality calculated from statistics recieved from api
+ */
+export const getDataQuality = (
+  statistics: Record<string, IGeneralStatistics>,
+  columnList: IHeaderNamesList[]
+) => {
+  const dataQuality: Array<Record<string, string | number>> = [];
+  columnList.forEach((columnName: IHeaderNamesList) => {
+    const generalValues: Record<string, string | number> = statistics[columnName.name].general;
+    if (generalValues.null) {
+      const nullCount = generalValues.null || 0;
+      dataQuality.push({
+        label: columnName.name,
+        value: nullCount,
+      });
+    } else {
+      dataQuality.push({
+        label: columnName.name,
+        value: '0',
+      });
+    }
+  });
+  return dataQuality;
+};
+
+const getProgressBarWrapperComponent = (dataQualityPercentValue) => {
+  return dataQualityPercentValue === 0 ? GreenSemiCircleProgressBar : RedSemiCircleProgressBar;
+};
+
+export default function DataQualityCircularProgressBar({
+  dataQualityPercentValue,
+}: {
+  dataQualityPercentValue: number;
+}) {
+  const Wrapper = getProgressBarWrapperComponent(dataQualityPercentValue);
+
+  return (
+    <Wrapper data-testid="semi-circular-progress-bar-wrapper">
+      <SemiCircleProgressBar
```

Comment:
please talk with the designer - he made a mock up of a simple line progress bar and i think it looks much better/fits the rest of the product more.